public/index.html

```diff
@@ -2,14 +2,17 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
```

Comment:
The links you are not using can be removed rather than commented out. I don't envision your app needing access to the React icons in the future.